src/test/regress/.gitignore

```diff
@@ -38,7 +38,6 @@ create_function_2.sql
 largeobject.sql
 misc.sql
 security_label.sql
-tablespace.sql
```

Comment:
Postgres Vanilla tests also include a test with the same name. Hence we had gitignore rules for this file in 2 locations.

The best solution I can think of is renaming this to `multi_tablespaces` so that we avoid name collisions. If this looks like a good idea, I can open a separate PR that addresses this.